engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content-hosts-helper.service.js

```diff
@@ -23,10 +23,6 @@ angular.module('Bastion.content-hosts').service('ContentHostsHelper',
         this.getHostStatusIcon = function (globalStatus) {
             var icons;
             var colors = {
-                // we can remove 'valid', 'partial', and 'invalid' when http://projects.theforeman.org/issues/15347 is fixed
```

Comment:
It seemed like a good time to respect this comment, so I removed the code. It looks like this helper function receives only integer inputs these days (based on the host status from the API).